features/execution_order.feature

```diff
@@ -74,13 +74,13 @@ Feature: Execution order
       #}
 
       """
-    Given I set the environment variables to:
+    And I set the environment variables to:
```

Comment:
This change likely doesn't require updating the Dredd hooks repositories, it is only refactoring (and only has a minor impact on readability). I think the linting rule is good to have however, and this gave me the opportunity to check that the build was breaking as expected and the corresponding output was [readable](https://travis-ci.org/gonzalo-bulnes/dredd-hooks-template/builds/193999105#L155).